we don't link against libgnome, but we do gtk, so fix the pkg-config call

--- a/configure.ac
+++ b/configure.ac
@@ -148,7 +148,7 @@ AC_HELP_STRING([--with-gtk],
 )
 
 if test "x$with_gtk" = "xyes"; then
- PKG_CHECK_MODULES([GSKI], [libglade-2.0 libgnomeui-2.0 >= 1.110.0])
+ PKG_CHECK_MODULES([GSKI], [libglade-2.0 gtk+-2.0])
  AC_DEFINE(HAVE_GTK, 1, [define if you have GTK support])
 fi
 AC_SUBST(GSKI_CFLAGS)
